frontend/src/modules/machinery/components/TranslationSource.js

```diff
@@ -0,0 +1,47 @@
+/* @flow */
+
+import React from 'react';
+
+import type { MachineryTranslation } from 'core/api';
+
+import GoogleTranslation from './GoogleTranslation';
+import MicrosoftTranslation from './MicrosoftTranslation';
+import MicrosoftTerminology from './MicrosoftTerminology';
+import TransvisionMemory from './TransvisionMemory';
+import CaighdeanTranslation from './CaighdeanTranslation';
+import TranslationMemory from './TranslationMemory';
+
+
+type Props = {|
+    translation: MachineryTranslation,
+|};
+
+
+/**
+ * Shows a list of translation sources.
+ */
+export default function TranslationSource(props: Props) {
+        const {translation} = props;
```

Comment:
Please add spaces between brackets for consistency with the rest of the code base.